### PR TITLE
CORE-3808: nvme: secure erase

### PR DESCRIPTION
**Pull request by shahar-lev**

### CODE REVIEW

File: hw/block/nvme.c [hunk 1]
```diff
@@ -1671,6 +1667,16 @@ static uint16_t nvme_format(NvmeCtrl *n, NvmeCmd *cmd)
     uint8_t pi = (dw10 >> 5) & 0x7;
     uint8_t sec_erase = (dw10 >> 8) & 0x7;
 
+    if (sec_erase) {
+        /* FIXME: BAD! ignores namespaces */
```
Comment (barakp) [inline hw/block/nvme.c]: ha?

File: hw/block/nvme.c [hunk 2]
```diff
@@ -1671,6 +1667,16 @@ static uint16_t nvme_format(NvmeCtrl *n, NvmeCmd *cmd)
...
+        if (bdrv_make_zero(n->conf.bs, 0)) {
```
Comment (barakp) [inline hw/block/nvme.c]: remove brackets (here and below)

Comment (shahar-lev) [inline hw/block/nvme.c]: ack

File: hw/block/nvme.c (hunk 1 above)
Comment (shahar-lev) [inline hw/block/nvme.c]: If I'm not mistaken, if one uses multiple namespaces then we can't zero out the entire block device (cause we'll nullify data from all namespaces?)

**Pull request closed without merging**